pkg/virtctl/vnc/vnc.go

```diff
@@ -31,31 +31,29 @@ import (
 	flag "github.com/spf13/pflag"
 	kubev1 "k8s.io/api/core/v1"
 
-	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/pkg/virtctl"
 )
 
 const FLAG = "vnc"
 
 type VNC struct{}
 
 func (o *VNC) Run(flags *flag.FlagSet) int {
-	server, _ := flags.GetString("server")
-	kubeconfig, _ := flags.GetString("kubeconfig")
 	namespace, _ := flags.GetString("namespace")
 	if namespace == "" {
 		namespace = kubev1.NamespaceDefault
 	}
 
 	if len(flags.Args()) != 2 {
 		log.Println("VM name is missing")
-		return 1
+		return virtctl.STATUS_ERROR
 	}
 	vm := flags.Arg(1)
 
-	virtCli, err := kubecli.GetKubevirtClientFromFlags(server, kubeconfig)
+	virtCli, err := virtctl.GetKubevirtClient(flags)
```

Comment:
Why are you adding this `virtcl.GetKubevirtClientFromFlags`? `kubecli.GetKubeivirtClient` already takes `server` and `kubeconfig` flags into account.

Reply:
This `Run` function takes `flags` as a parameter--which is technically different than the default command line flags (especially in the test suite). master and kubeconfig from the flags passed into `Run` as a parameter should be honored.

This is simply a refactor to remove duplicate code between vnc.go, console.go, and offlinevm.go. Perhaps the real issue here is that `virtctl.GetKubevirtClient` should really be located in `kubecli` and we should think about why a function named `kubecli.GetKubevirtClientFromFlags` takes two strings as parameters. Perhaps it should be named `kubecli.GetKubevirtClientFromConfig`?

Reply:
I was refactoring the code to match my previous suggestion and realized: virtctl uses a flag called `server` while kubecli uses one called `master`...

Because this has direct client API implications we shouldn't change this capriciously, but shouldn't those be unified?

Reply:
Please check what kubectl uses and adjust our code appropriately.